databricks_cli/sdk/api_client.py

```diff
@@ -127,3 +128,13 @@ def perform_query(self, method, path, data = {}, headers = None):
                 pass
             raise requests.exceptions.HTTPError(message, response=e.response)
         return resp.json()
+
+
+def _translate_boolean_to_query_param(value):
+    assert not isinstance(value, list), 'The value of the data dict cannot be a list'
```

Comment:
This is a weird error, referring to `data dict`. Maybe "GET parameters cannot pass lists of objects" or something?